From: Zhang Chujun <zhangchujun@cmss.chinamobile.com>
Subject: selftest/vm_util: remove unused variables
Date: Tue, 4 Nov 2025 14:28:55 +0800

The global variables __page_size and __page_shift are declared in
vm_util.c, but they are never used anywhere in the codebase.  This patch
removes them to clean up the source and eliminate dead code.

Link: https://lkml.kernel.org/r/20251104062855.2468-1-zhangchujun@cmss.chinamobile.com
Signed-off-by: Zhang Chujun <zhangchujun@cmss.chinamobile.com>
Acked-by: David Hildenbrand (Red Hat) <david@kernel.org>
Cc: Thomas Weißschuh <thomas.weissschuh@linutronix.de>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/mm/vm_util.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/tools/testing/selftests/mm/vm_util.c~selftest-vm_util-remove-unused-variables
+++ a/tools/testing/selftests/mm/vm_util.c
@@ -17,9 +17,6 @@
 #define STATUS_FILE_PATH "/proc/self/status"
 #define MAX_LINE_LENGTH 500
 
-unsigned int __page_size;
-unsigned int __page_shift;
-
 uint64_t pagemap_get_entry(int fd, char *start)
 {
 	const unsigned long pfn = (unsigned long)start / getpagesize();
_
